pipeline/testing/average_rating_test.py

```diff
@@ -0,0 +1,91 @@
+import sys
+import unittest
+
+from average_rating import calculate_analytics
+
+sys.path.insert(0, "/workspace/butterfly/pipeline/scripts/")
```

Comment:
This part does need to change because the tests are failing due to this import not working correctly. Change it to this and then the tests will pass and you will be allowed to merge:

```python
import unittest

from pipeline.scripts.average_rating import calculate_analytics
```